DigitalLearningSolutions.Web/Views/LearningContent/_BrandCourseCard.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningContent
+@model BrandCourseViewModel
+
+<div class="searchable-element nhsuk-panel card-with-buttons word-break nhsuk-u-margin-bottom-4" id="@Model.ApplicationId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.ApplicationId-name" name="course-name">
+        @Model.ApplicationName
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <div class="tags">
+        <div class="card-filter-tag">
+          <strong class="nhsuk-tag nhsuk-tag--grey">@Model.DisplayTime</strong>
+          <span hidden name="learning-time">@Model.Time</span>
+        </div>
+
+        <div class="card-filter-tag" @(Model.PopularityRating <= 0.5 ? "hidden" : string.Empty)>
+          <strong class="nhsuk-tag nhsuk-tag--green">Popular</strong>
+          <span hidden name="popularity-score">@Model.PopularityRating</span>
+        </div>
+      </div>
+      <span hidden data-filter-value="@Model.CategoryFilter">
+        @Model.CategoryName
+      </span>
+      <span hidden data-filter-value="@Model.TopicFilter">
+        @Model.CourseTopic
+      </span>
+      <span hidden name="created-date">
+        @Model.CreatedDate
+      </span>

```

Comment:
Feels a bit weird to be sorting by something that is hidden. Do we want to display this instead?

Also could use "data-name-for-sorting" as the attribute for the sortable name.

Reply:
Confirmed with Kevin, we want to add this as visible